PluralKit.Bot/Commands/Misc.cs

```diff
@@ -214,9 +228,29 @@ public async Task PermCheckGuild(Context ctx)
                     eb.Color(DiscordUtils.Red);
                 }
             }
+            string footer = "";
+
+            foreach (var (missingPermissionField, channels) in permissionsMissing)
+            {
+                if (missingPermissionField == (ulong)PermissionSet.UseExternalEmojis && footer.Length < 1)
+                {
+                    footer = "Use External Emojis permissions must be granted to the @everyone/default role.";
+                }
+            }
 
             if (hiddenChannels > 0)
-                eb.Footer(new($"{"channel".ToQuantity(hiddenChannels)} were ignored as you do not have view access to them."));
+            {   
+                if (footer.Length > 0)
+                {
+                    footer = $"{"channel".ToQuantity(hiddenChannels)} were ignored as you do not have view access to them.";
+                }
+                else
+                {
+                    footer += $"\n{"channel".ToQuantity(hiddenChannels)} were ignored as you do not have view access to them.";
```

Comment:
I don't think you can use newlines in an embed footer? either way, this should not be duplicated - maybe do something like
```cs
if (footer.Length > 0)
    footer += "\n";

footer += $"[...]";
```